test/e2e/specs/plugins/plugins__search.ts

```diff
@@ -20,6 +20,7 @@ describe( DataHelper.createSuiteTitle( 'Plugins search' ), function () {
 	it( 'Visit plugins page', async function () {
 		pluginsPage = new PluginsPage( page );
 		await pluginsPage.visit();
+		await page.waitForLoadState( 'networkidle' );
```

Comment:
This is needed because the search input gets defocused when the SSR version of the content gets switched with the client one.

Reply:
**question**: Instead of adding a floating `waitForLoadState('networkidle')` in the spec itself, would it make sense to apply this wait directly in the `pluginsPage.visit` method?